Adiciona translucidez à barra de status/Altera mensagem no início

vaporwavie opened the pull request:
**Esse PR precisa ser testado**
Meu notebook tá tenso, travou na criação do apk. não consegui testar 100% a feature :disappointed: 

Os detalhes dessa feature foram adicionados no commit [d52f6f1](https://github.com/vaporwavie/DeVryCalc/commit/d52f6f162d75d9d22bc4da822b5d08298cb6af58#commitcomment-20805348)

vaporwavie committed:
Adiciona (materializa) cor à barra de status

vaporwavie committed:
Merge branch 'master' of github.com:vaporwavie/DeVryCalc

vaporwavie committed:
Altera mensagem sugerindo adição de disciplinas

RodolfoSilva commented:
Man, fiz o teste aqui e funcionou :smile: .

Só tem um porém, eu olhei a cor usada por outros aplicativos e na indicação é que a barra de status seja um pouco mais escura que a toolbar.

Então a cor que eu gerei com base na cor da toolbar só que 10% mais escura foi essa aqui: `#00375f`.

Olha como ficou:
![status-bar](https://cloud.githubusercontent.com/assets/1703020/22758698/94cabb76-ee2e-11e6-9cff-7d87b24c6c08.jpeg)

Fora isso, ficou perfeito.

vaporwavie commented:
Massa! Que bom que funcionou... Queria poder ter testado antes de mandar, mas já foi 10/10 saber que tudo foi implementado corretamente :D
Ah foi, esqueci de separar o PR da mudança de texto no início (aquele <code>p</code>), mas acabou indo junto auhauha

vaporwavie committed:
Atualiza coloração da barra de status

Sugestão de @RodolfoSilva no PR [#5](https://github.com/RodolfoSilva/DeVryCalc/pull/5)
